Makefile

```diff
@@ -61,7 +64,21 @@ trustzone: sdk trustzone-env
 	$(MAKE) -C trustzone-root-enclave trustzone OPTEE_DIR=$(OPTEE_DIR) OPTEE_OS_DIR=$(OPTEE_OS_DIR)
 	cd veracruz-client && RUSTFLAGS=$(SGX_RUST_FLAG) cargo build --lib --features tz
 
-sgx-veracruz-server-test:
+# Using wildcard in the dependencies because if they are there, and newer, it
+# should be rebuilt, but if they aren't there, they don't need to be built 
+# (they are optional)
+veracruz-test/proxy-attestation-server.db: #$(wildcard sgx-root-enclave/css.bin) $(wildcard nitro-root-enclave/PCR0)
```

Comment:
Are the dependencies supposed to be commented out here?

Reply:
They are not. Will fix now